app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
+native `log.Printf()`.
+  - Introduced support for multiple pluginservers. This feature is backward-compatible with
+  the existing single Go pluginserver.
+
+#### Plugin Development Kit (PDK)
+- Introduced a `kong.node.get_hostname` method that returns current node's host name.
+- Introduced a `kong.cluster.get_id` method that returns a unique ID
+  for the current {{site.base_gateway}} cluster. If the gateway is running in DB-less mode
+  without a cluster ID explicitly defined, this method returns nil.
+  For Hybrid mode, all Control Planes and Data Planes belonging to the
+  same cluster returns the same cluster ID. For traditional database
+  type deployments, all {{site.base_gateway}} nodes pointing to the same database will
+  also return the same cluster ID.
+- Introduced a `kong.log.set_serialize_value`, which allows customizing
+  the output of `kong.log.serialize`.
+
+#### Plugins
+- The [HTTP Log](https://docs.konghq.com/hub/kong-inc/http-log/) (`http-log`) plugin
+now lets you add a table of headers to the HTTP request using the
+`headers` configuration parameter, which will help you integrate with many observability systems.
+- The [Key Authentication](https://docs.konghq.com/hub/kong-inc/key-auth/) (`key-auth`)
+plugin has two new configuration parameters: `key_in_header` and `key_in_query`. Both
+are booleans and tell {{site.base_gateway}} whether to accept (true) or reject (false) passed in either
+the header or the query string. Both default to “true.”
+- The [Request Size Limiting](https://docs.konghq.com/hub/kong-inc/request-size-limiting/)
+(`request-size-limiting`) plugin has a new configuration `require_content_length` that
+causes the plugin to ensure a valid Content-Length header exists before reading the request body.
+- The [Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/)
+(`serverless-functions`) plugin introduces a sandboxing capability, is *enabled* by default,
+and where only Kong PDK, OpenResty `ngx` APIs, and Lua standard libraries are allowed.
+- The [LDAP Authentication Advanced](https://docs.konghq.com/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`) plugin has two new features:
+  - added config `log_search_results` that allows displaying all of the LDAP search results received from the LDAP server.
+  - additional debug log statements added for authenticated groups.
+- jq plugin
+- Rate Limiting Advanced:
+  - add a jitter (random delay) for the Retry-After header
+- mtls-auth
+  - add support for tags in the DAO
+  - plugin ensures the existence of any provided CAs when creating the plugin entry
+- Response transformer advanced:
+  - json paths for nested elements and arrays:
+  - transform gzipped content:
+- Collector
+  - Support for hybrid mode
+  - remove log_bodies configuration option
+- OpenID Connect Library
+  - Token introspection now checks the status code properly
```

Comment:
What happened when it didn't check the status code properly?  How would a user know they were susceptible to this issue?